inspect data command broken

The `eris data inspect` command doesn't really work.

With 0 arguments, it prints out the help screen.

With 1 or more arguments it crashes:

```
$ eris data inspect ipfs Config.Image
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/eris-ltd/eris-cli/data.InspectData(0xc8200ced80, 0x0, 0x0)
...
```

--- a/commands/data.go
+++ b/commands/data.go
@@ -149,14 +149,15 @@ func RenameData(cmd *cobra.Command, args []string) {
 }
 
 func InspectData(cmd *cobra.Command, args []string) {
-	IfExit(ArgCheck(2, "ge", cmd, args))
-
-	if len(args) == 1 {
-		args = append(args, "all")
-	}
+	IfExit(ArgCheck(1, "ge", cmd, args))
 
 	do.Name = args[0]
-	do.Path = args[1]
+	if len(args) == 1 {
+		do.Args = []string{"all"}
+	} else {
+		do.Args = []string{args[1]}
+	}
+
 	IfExit(data.InspectData(do))
 }
 

